### PR TITLE
added clamav to production

### LINKED ISSUE #246: Add virus checking
Need to add virus checking by adding clamav.
https://github.com/projecthydra/curation_concerns#virus-detection

### PR DESCRIPTION
**Pull request by sethaj**
Resolved #246

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 language: ruby
 cache: bundler
-sudo: false
+sudo: required
 
+before_install:
```
Comment (jmcglone) [inline .travis.yml]: If clamav install ends up slowing Travis down significantly, we may want to exclude it from this process. I think there is a way to still include it in the Gemfile but when given a `TRAVIS` environment variable, clamav would not get built.  See how Scholarsphere does it: https://github.com/psu-stewardship/scholarsphere/blob/f4e46d132c2042d6cad73115773a179f0f56245e/Gemfile#L19

**Comment by jmcglone** [comment]
@sethaj another question -- do we know happens when a file gets uploaded that is rejected by clamav?

File: .travis.yml (hunk 1 above)
Comment (sethaj) [inline .travis.yml]: Interesting, I hadn't seen that. Agree that if it slows down tests we should exclude it.

**Comment by sethaj** [comment]
@jmcglone Looks like it happens in Hydra::Works, https://github.com/projecthydra/hydra-works/blob/master/lib/hydra/works/virus_scanner.rb

**Pull request merged**